app/controllers/TrusteesUkAddressController.scala

```diff
@@ -77,8 +77,10 @@ class TrusteesUkAddressController @Inject()(
           Future.successful(BadRequest(view(formWithErrors, mode, index, trusteeName))),
 
         value => {
+          val transformedPostcode = value.postcode.toUpperCase()
```

Comment:
Can we do this as part of the Mapping `postcode()` type? We already trim the postcode there.

Reply:
Done.